worker/src/RTC/RtpPacket.cpp

```diff
@@ -121,21 +121,10 @@ namespace RTC
 		           payloadLength + size_t{ payloadPadding },
 		  "packet's computed size does not match received size");
 
-		auto* packet = RtpPacketPool.Allocate();
-		new (packet) RtpPacket(header, headerExtension, payload, payloadLength, payloadPadding, len);
+		auto* packet =
+		  new RtpPacket(header, headerExtension, payload, payloadLength, payloadPadding, len);
```

Comment:
`new` like this means heap allocation. Previous construction was used to avoid this.

Reply:
I know, yes.

This PR fixes an issue in XXXTransport.cpp where a shared pointer (returned by `RtpPacket::Parse()`) was being misused driving to undefined behaviour.

About this `new`. We can use instead the memory from the pool here, but how to deallocate it?

Reply:
I'll give it a try.

Reply:
Done